scripts/release_npm.sh

```diff
@@ -33,13 +33,19 @@ fi
 
 # Command to push the packages
 CMD_UPDATE_VERSION="lerna version ${PACKAGE_VERSION} --yes --exact --force-publish --no-git-tag-version --no-push"
-CMD_PUBLISH_PACKAGES="lerna publish ${PACKAGE_VERSION} ${TAG} --yes --exact --force-publish --no-git-tag-version --no-push --no-verify-access --no-verify-registry"
+CMD_PUBLISH_PACKAGES="lerna publish ${PACKAGE_VERSION} ${TAG} --yes --force-publish --no-git-tag-version --no-push --no-verify-access"
 
-# Publish the packages to npm. Note that lerna cleans the working tree after this as of 3.0.4, so we need to reapply version
-# https://github.com/lerna/lerna/blob/3cbeeabcb443d9415bb86c4539652b85cd7b4025/commands/publish/index.js#L354-L363
-echo $CMD_PUBLISH_PACKAGES;
-$CMD_PUBLISH_PACKAGES;
-
-# Update package version again for later commit during the "commit release" stage
+# Update package version and build. This ensure the dist files are generated with the version that will be released.
+# The publish step will only publish, it does not build and generate the files with the provided version.
 echo $CMD_UPDATE_VERSION;
 $CMD_UPDATE_VERSION;
+
+# Build to regenerate distribution files with new package version, commit the version changes
+echo "Building artifacts for v${PACKAGE_VERSION} and creating release commit"
+yarn build;
+git add CHANGELOG.md lerna.json packages/*;
+git commit -m "release: v${PACKAGE_VERSION}";
```

Comment:
Where is the associated `git push`? 
The commit should also be tagged with the released version.

Reply:
It is in our internal ci script(lwc/ci-integration)